be/src/vec/columns/column_string.cpp

```diff
@@ -777,6 +777,14 @@ void ColumnStr<T>::insert(const Field& x) {
     sanity_check_simple();
 }
 
+template <typename T>
+bool ColumnStr<T>::is_ascii() const {
+    if (chars.empty()) {
```

Comment:
if empty() the `simd::VStringFunctions::is_ascii(StringRef(chars.data(), chars.size()));` also return true